watcher/lib/influxdb/rollups.go

```diff
@@ -153,10 +153,10 @@ func buildFunction(f Function) (string, error) {
 		if err != nil {
 			return "", trace.Wrap(err)
 		}
-		return fmt.Sprintf("%v(%v, %v) as %v", funcName, f.Field, param, alias), nil
+		return fmt.Sprintf("%v(\"%v\", %v) as %v", funcName, f.Field, param, alias), nil
```

Comment:
I would just use backticks for the string itself if you need to use quotes inside it.

```go
fmt.Sprintf(`%v("%v", %v) as %v`, ...
```

Reply:
Fixed.